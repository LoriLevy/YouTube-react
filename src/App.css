.App-logo {
  animation: App-logo-spin infinite 20s linear;
  height: 80px;
}

.App-header {
  background-color: #222;
  height: 150px;
  padding: 20px;
  color: white;
}

.App-intro {
  font-size: large;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }

  to {
    transform: rotate(360deg);
  }
}


body {
  background-color: #2980b9;
}

.youtube {
  border: 3px #2c3e50 dashed;
  margin-top: 20px;
}

button {
  margin: 10px;
  padding: 10px 20px;
  font-size: 20px;
  background-color: #e67e22;
}

div {
  color: white;
}