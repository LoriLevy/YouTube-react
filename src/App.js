import React, {Component} from 'react';
import {Home} from './containers/Home/Home';
import {Watch} from './containers/Watch/Watch';
import {AppLayout} from './components/AppLayout/AppLayout';
import {Route, Switch} from 'react-router-dom';

class App extends Component {
  render() {
    return (
      <AppLayout>
        <Switch>
          <Route path="/Watch" component={Watch}/>
          <Route path="/" component={Home}/>
        </Switch>
      </AppLayout>
    );
  }
}

export default App;