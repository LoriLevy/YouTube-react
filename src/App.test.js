import React from 'react';
import {shallow} from 'enzyme';
import App from './App';

// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });

describe('App', () => {
  test('renders', () => {
    const wrapper = shallow(
      <App/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
