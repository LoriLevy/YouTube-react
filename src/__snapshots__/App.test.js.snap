// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<AppLayout>
  <Switch>
    <Route
      component={[Function]}
      path="/Watch"
    />
    <Route
      component={[Function]}
      path="/"
    />
  </Switch>
</AppLayout>
`;
